pkg/installer/generateconfig.go

```diff
@@ -31,6 +32,18 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
+func supportedDefaultDisk(vmSku *mgmtcompute.ResourceSku) string {
```

Comment:
I'm wondering if it would be better to move this entire function and its tests to the `computeskus` package instead of having it here. This is currently the only place we're using it, but if there's anything else that arises from supported disk types in the future it would be beneficial to have it available there.

Reply:
+1 on moving ResourceSku stuff to `computeskus`.